test/tests_for_integration/api_to_db_test.py

```diff
@@ -84,12 +104,65 @@ def auth_url(config, mongo_client):
     _clean_auth_db(mongo_client)
 
 
-# TODO start the ee2 service
-# TODO wipe the ee2 database between every test
+# side effect: modifies the config
+def _create_config_file(full_config, auth_url):
+    ee2c = full_config[EE2_CONFIG_SECTION]
```

Comment:
why not use `get_ee2_test_config()`? It isn't clear why there is a dedicated method for getting the EE2 config section but you're not using it here.